theblues/jem.py

```diff
@@ -27,7 +31,13 @@ def fetch_macaroon(self):
             # We don't use make_request b/c we don't want the request to be
             # fully handled. This lets us get the macaroon out of the request
             # and keep it.
-            response = requests.get("{}model".format(self.url))
+            url = "{}model".format(self.url)
+            response = requests.get(url)
```

Comment:
Doesn't the timeout need to be specified here?  request.get(url, timeout=self.timeout) ?